### ISSUE TITLE
Suggestion: Allow Export from YYMMDD to YYMMDD instead of YYYY to YYYY

### ISSUE THREAD

**Issue by dsnvwlmnt** [arty-name/livejournal-export#22]
Currently this tool is mainly useful for a one-time download of a blog. You can only download entire years at a minimum.

It would be more generally useful to be able to make incremental backups as well. All that's needed for that is granularity of starting and ending date. Instead of downloading an entire year at a minimum, adding Month and Day would allow granular backups.

If this is not entirely possible because of the LiveJournal API, Month could still be added so incremental backups by month are possible instead of entire years.

**Comment by arty-name** [comment]
I agree with you that this would be more generally useful, and I welcome the PR to implement this. My use case for this tool was exactly one-time download of a blog when I left Livejournal 10 years ago, and I don’t actively work on it anymore, only reviewing PRs.

Would you like to create a PR?

**Comment by dsnvwlmnt** [comment]
Possibly. Never created one, would have to look into the whole process.

**Comment by arty-name** [comment]
Good luck! :four_leaf_clover: Please keep in mind that all of a sudden I have two contributors to the project at once, and there might be merge conflicts between the branches.

**Comment by arty-name** [comment]
The other PR is now merged, by the way

**Comment by arty-name** [comment]
Congratulations! A kind stranger has implemented this feature.
